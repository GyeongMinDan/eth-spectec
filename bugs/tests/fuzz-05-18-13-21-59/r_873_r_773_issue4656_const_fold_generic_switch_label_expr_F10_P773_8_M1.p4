     extern bit<8> baz();
        control c()() {
         apply {
          switch (baz()) {
       ((((((1) + (2))) == (3))) ? (1) : (2)): 
       ((((((1) + (2))) == (3))) ? (1) : (2)): 
     }
        }
       }
