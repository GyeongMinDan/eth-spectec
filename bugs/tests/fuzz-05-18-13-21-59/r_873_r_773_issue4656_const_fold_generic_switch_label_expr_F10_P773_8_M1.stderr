likely_hits/r_873_r_773_issue4656_const_fold_generic_switch_label_expr_F10_P773_8_M1.p4(6): [--Wwarn=missing] warning: SwitchCase: fallthrough with no statement
       ((((((1) + (2))) == (3))) ? (1) : (2)):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
likely_hits/r_873_r_773_issue4656_const_fold_generic_switch_label_expr_F10_P773_8_M1.p4(2): [--Wwarn=unused] warning: Control c is not used; removing
        control c()() {
                ^
[--Wwarn=missing] warning: Program does not contain a `main' module
